eng/scripts/update-pathversions.rs

```diff
@@ -10,14 +10,24 @@ toml_edit = "^0.22"
 ---
 
 
+
```

Comment:
Extreme nit: so

much



whitespace.

Reply:
I didn't even notice that